Podcast/EpisodeUtilityButtonBarView.swift

```diff
@@ -121,11 +130,18 @@ class EpisodeUtilityButtonBarView: UIView {
     }
 
     func setupWithEpisode(episode: Episode) {
-        playButton.isSelected = episode.isPlaying
         slider.setSliderProgress(isPlaying: episode.isPlaying, progress: episode.currentProgress)
         bookmarkButton.isSelected = episode.isBookmarked
         recommendedButton.setupWithNumber(isSelected: episode.isRecommended, numberOf: episode.numberOfRecommendations)
         topLineseparator.isHidden = !slider.isHidden
         downloaded.setupWith(episode: episode)
+        playButton.configure(for: episode)
+        bookmarkButton.isEnabled = episode.audioURL != nil
+        bookmarkButton.isHidden = episode.audioURL == nil
+        recommendedButton.isEnabled = episode.audioURL != nil
+        recommendedButton.isHidden = episode.audioURL == nil
+        greyedOutLabel.isHidden = episode.audioURL != nil
+        moreButton.isEnabled = episode.audioURL != nil
+        moreButton.isHidden = episode.audioURL == nil
```

Comment:
If it's hidden is it still enabled?

Reply:
oh true if it's hidden its not, i'll fix that right now